client/package.json

```diff
@@ -43,6 +43,7 @@
     "webpack": "^1.10.5"
   },
   "devDependencies": {
+    "es6-promise": "^2.3.0",
```

Comment:
right, but not sure how to get es6-promise to be directly under node_modules without doing this (see original PR description)